DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv4/test_ipv4_mtu.py

```diff
@@ -0,0 +1,212 @@
+import asyncio
+import pytest
+
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.lib.ip.ip_address import IpAddress
+from dent_os_testbed.lib.ethtool.ethtool import Ethtool
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_setup_streams,
+    tgen_utils_start_traffic,
+    tgen_utils_stop_traffic,
+    tgen_utils_get_loss,
+)
+
+pytestmark = [
+    pytest.mark.suite_functional_ipv4,
+    pytest.mark.usefixtures("cleanup_ip_addrs", "cleanup_tgen", "enable_ipv4_forwarding"),
+    pytest.mark.asyncio,
+]
+
+
+async def get_port_stats(dent, ports):
+    stats = {}
+    for port in ports:
+        out = await Ethtool.show(input_data=[{dent: [
+            {"devname": port, "options": "-S"}
+        ]}], parse_output=True)
+        assert out[0][dent]["rc"] == 0
+        stats[port] = out[0][dent]["parsed_output"]
+    return stats
+
+
+@pytest.mark.usefixtures("change_port_mtu")
+async def test_ipv4_oversized_mtu(testbed):
+    """
+    Test Name: test_ipv4_oversized_mtu
+    Test Suite: suite_functional_ipv4
+    Test Overview: Test IPv4 oversized mtu counters
+    Test Procedure:
+    1. Init interfaces
+    2. Configure ports up
+    3. Configure IP addrs
+    4. Configure interfaces MTU to 1000
+    5. Generate traffic with packet size 1200 and verify there's no reception due to MTU
+    6. Verify oversized counter been incremented in port statistics
+    """
+    # 1. Init interfaces
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
```

Comment:
This check is done in the change_port_mtu

Reply:
If the fixture fails it will not fail the test. So this check is still necessary.

Reply:
Maybe then just returning is not the correct way to process this situation? Probably skip is better. If we just return, the test has a PASS result, but in fact, it isn't executed at all. So let's change to skip in all places.

Reply:
Then I suggest to do this in a separate PR